packages/mobile/src/screens/edit-track-screen/components/SpecialAccessAvailability.tsx

```diff
@@ -107,19 +107,6 @@ export const SpecialAccessAvailability = ({
     : neutral
 
   const { set: setTrackAvailabilityFields } = useSetTrackAvailabilityFields()
-
-  useEffect(() => {
-    if (selected) {
-      setTrackAvailabilityFields(
-        {
-          is_premium: true,
-          premium_conditions: { follow_user_id: 1 }
-        },
-        true
-      )
-    }
-  }, [selected, setTrackAvailabilityFields])
-
   const [{ value: premiumConditions }, , { setValue: setPremiumConditions }] =
     useField<Nullable<PremiumConditions>>('premium_conditions')
   const isFollowerGated = useMemo(() => {
```

Comment:
This stuff doesn't need to be `memo`'d since it's just checking for the existence of a field